scripts/core/EXP-COMMAND-BUFFER.rst

```diff
@@ -77,9 +77,12 @@ Command-Buffer Creation
 Command-Buffers are tied to a specific ${x}_context_handle_t and
 ${x}_device_handle_t. ${x}CommandBufferCreateExp optionally takes a descriptor
 to provide additional properties for how the command-buffer should be
-constructed. The only unique member defined in ${x}_exp_command_buffer_desc_t
-is ``isUpdatable``, which should be set to ``true`` to support :ref:`updating
+constructed. The members defined in ${x}_exp_command_buffer_desc_t are:
+* ``isUpdatable``, which should be set to ``true`` to support :ref:`updating
 command-buffer commands`.
+* ``isInOrder``, which should be set to ``true`` to enable the graph workload
+  to be submitted to an in-order command-list.
+* ``enableProfiling``, which should be set to ``true`` to enable graph profling.
```

Comment:
Think we could improve the language here a bit:
* `graph workload` - We probably shouldn't be talking about graphs at this abstraction layer, would say "command-buffer" workload.
* `command-list` - "command-list" is L0 terminology, should be in terms of "command-buffer".
* `graph profling.` - "profiling of the command-buffer"